### PR TITLE
#13416 standardize endpoint quickchart

### PR DESCRIPTION
**Pull request by angel-perez-itti**
Pull request for the topic https://github.com/slingr-stack/platform/issues/13416 created by @pasaperez-slingr 

## What does it do?

This ticket implements the use of slingr-helpgen, whereby it modifies existing functions, adds basic functions and a flow step.

## How to test it?

Make calls from the platform for example using the console using the information provided in the documentation.
```js
app.endpoints.quickchart.chart.post({
        "backgroundColor": "white",
        "width": "500",
        "height": "300",
        "format": "png",
        "chart": {
          "type": "bar",
          "data": {
            "labels": [2012, 2013, 2014, 2015, 2016],
            "datasets": [{
              "label": "labelA",
              "data": [12, 6, 5, 18, 12]
            }, {
              "label": "labelB",
              "data": [4, 8, 16, 5, 5]
            }]
          }
      });
```

```js
app.endpoints.quickchart.qr.get({
    "text": "description",
    "dark": "f23",
    "light": "55f",
    "ecLevel": "Q",
    "size": "300"
  });
```

## Implementation notes

Create a new tags changing the major version of the endpoint, it's partially backward compatible.

### CODE REVIEW

**Comment by sebasmoyano** [comment]
#approve

File: README.md
Comment (agreggio-slingr) [inline README.md:1]: Add to Readme the documentation created by the helperGen and the Flows Steps added.

File: flowSteps/generateChartQuickchart/step.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+/**
+ * This flow step will send a generic request to generate a Chart.
+ *
+ * @param {object} inputs
+ */
+step.generateChartQuickchart = function (inputs) {
+
+	sys.logs.warn(JSON.stringify(inputs));
+	var inputsLogic = {
```
Comment (agreggio-slingr) [inline flowSteps/generateChartQuickchart/step.js]: I do not think this log is necessary

File: flowSteps/generateQRQuickchart/step.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/**
+ * This flow step will send a generic request to generate QR.
+ *
+ * @param {object} inputs
+ */
+step.generateQRQuickchart = function (inputs) {
+
+	sys.logs.warn(JSON.stringify(inputs));
+	var inputsLogic = {
```
Comment (agreggio-slingr) [inline flowSteps/generateQRQuickchart/step.js]: I do not think this log is necessary

File: README.md
Comment (pasaperez-work) [reply README.md:1]: Done

File: flowSteps/generateChartQuickchart/step.js (hunk 1 above)
Comment (pasaperez-work) [reply flowSteps/generateChartQuickchart/step.js]: Agreed, useful, but unnecessary.

File: flowSteps/generateQRQuickchart/step.js (hunk 2 above)
Comment (pasaperez-work) [reply flowSteps/generateQRQuickchart/step.js]: Agreed, useful, but unnecessary.

**Comment by agreggio-slingr** [comment]
#approve

**Pull request merged**